catalogmanagementv1/catalog_management_v1_examples_test.go

```diff
@@ -1774,6 +1976,17 @@ var _ = Describe(`CatalogManagementV1 Examples Tests`, func() {
 			Expect(response.StatusCode).To(Equal(200))
 		})
 
+		// unset pc managed
```

Comment:
I know this is really-really small, but please change to make it sync with the comment in the integration tests. :)
```suggestion
		// Unset pc managed
```